app/controllers/explorer_stake_accounts_controller.rb

```diff
@@ -19,6 +19,9 @@ def index
     end
 
     @explorer_stake_accounts = @explorer_stake_accounts[:explorer_stake_accounts]
+    @stake_history = ExplorerStakeAccountHistoryStat.where(network: index_params[:network])
```

Comment:
There are multiple records in this variable. It should be called @stake_histories, or @stake_histories_stats, or similar